airgun/views/ansible_variable.py

```diff
@@ -0,0 +1,116 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Select
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SatTable
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import CustomParameter
+from airgun.widgets import FilteredDropdown
+from airgun.widgets import Pagination
+from airgun.widgets import Pf4ConfirmationDialog
+from airgun.widgets import SatSelect
+
+
+class AnsibleVariablesView(BaseLoggedInView, SearchableViewMixin):
+    """Main Ansible Variables view"""
+
+    title = Text("//h1[contains(., text()='Ansible Variables')")
+    new_variable = Text("//a[contains(@href, '/ansible/ansible_variables/new')]")
+    total_variables = Text("//span[@class='pagination-pf-items-total']")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text("./a"),
+            'Role': Text("./a"),
+            'Type': Text("./a"),
+            'Imported?': Text("./a"),
+            'Actions': Text(".//a[@data-method='delete']"),
+        },
+    )
+    pagination = Pagination()
+    confirm = Pf4ConfirmationDialog()
+
+    @property
+    def is_displayed(self):
+        return self.title.is_displayed and self.new_variable.is_displayed
+
+
+class MatcherTable(CustomParameter):
+    add_new_value = Text("..//a[contains(text(),'+ Add Matcher')]")
+
+
+class MatcherActions(View):
+    """Interface table has Attribute type column that contains select field and
+    text input field."""
+
+    matcher_key = Select(".//select")
+    matcher_value = TextInput(locator=".//input[@class='matcher_value']")
```

Comment:
Ack, in the interest of getting this PR merged, I will leave this as is for now. If a future test requirement needs to access that `delete` button, it can be added at that time.